native/src/routes/intro/IntroContainer.js

```diff
@@ -10,24 +10,19 @@ import styled from 'styled-components/native'
 import { type StyledComponent } from 'styled-components'
 import AppSettings from '../../modules/settings/AppSettings'
 import SlideContent, { type SlideContentType } from './SlideContent'
-import initSentry from '../../modules/app/initSentry'
 import SlideFooter from './footer/SlideFooter'
 import type { ViewToken } from 'react-native/Libraries/Lists/ViewabilityHelper'
-import CustomizableIntroSettings from './CustomizableIntroSettings'
-import IntroSettings from './IntroSettings'
 import type { StateType as ReduxStateType } from '../../modules/app/StateType'
 import { connect } from 'react-redux'
-import { requestLocationPermission } from '../../modules/app/LocationPermissionManager'
 import buildConfig, { buildConfigAssets } from '../../modules/app/constants/buildConfig'
-import { requestPushNotificationPermission } from '../../modules/push-notifications/PushNotificationsManager'
 import type { NavigationPropType, RoutePropType } from '../../modules/app/constants/NavigationTypes'
 import { LANDING_ROUTE } from 'api-client/src/routes'
 import type { IntroRouteType } from 'api-client/src/routes'
 import navigateToDeepLink from '../../modules/navigation/navigateToDeepLink'
 import type { StoreActionType } from '../../modules/app/StoreActionType'
 
 const Container: StyledComponent<{| width: number |}, {||}, *> = styled.View`
-  display: flex;
+  flex: 1;
```

Comment:
I think the css here does not really make sense:
display: flex is the default for View, so it is completly fine to remove it, but
flex: 1, does make the container flexible, but this does not make sense here, cause we use height. So either we remove the flex stuff (flex, flex-direction and justify-content) or we remove the height: property, because both together is not working anyway. Flex wins over height setting.

Reply:
I always thought `flex: 1` is the same as `display: flex` :astonished: Good to know, thanks!